src/Microsoft.Framework.DesignTimeHost/ApplicationContext.cs

```diff
@@ -631,6 +636,33 @@ private void SendOutgoingMessages()
             }
         }
 
+        private void SendProjectFormatWarnings(IList<FileFormatWarning> projectFormatWarnings)
+        {
+            if (projectFormatWarnings.Count == 0)
+            {
+                return;
+            }
+
+            var message = new DiagnosticsMessage(
+                warnings: projectFormatWarnings.Select(warning => new DiagnosticsInfo
+                {
+                    Message = warning.Message,
+                    Line = warning.Line,
+                    Column = warning.Column,
+                    Path = warning.Path,
+                    FormattedMessage = warning.Message
+                }),
+                errors: null,
+                framework: null);
+
+            _initializedContext.Transmit(new Message
+            {
+                ContextId = Id,
+                MessageType = "Diagnostics",
```

Comment:
Maybe we should. But for now none of the `MessageType` value are given in const, but asall hard coded.